### PR TITLE
Feature/buildkite

### PR DESCRIPTION
**Pull request by GeorgeNash**
Work in progress - need a PR to test out Buildkite.

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ### Version 0.0.19 ###
 
+* Buildkite
```
Comment (migwellian) [inline release_notes.md]: should be under version 0.0.20 now

**Comment by wesleyw72** [comment]
PR now ready for review. Linux is setup to autoscale. There is currently one Windows agent, EV are working on making these autoscale and also allowing multiple agents per instance. We can manually request more Windows agents in the meantime though.

This is currently not setup to block PRs or even trigger on PRs. Once merged in, I will add the webhook to run it. After running with it for a while we can enable the merge blocking etc.

**Comment by wesleyw72** [comment]
PR now ready for review. Linux is setup to autoscale. There is currently one Windows agent, EV are working on making these autoscale and also allowing multiple agents per instance. We can manually request more Windows agents in the meantime though.

This is currently not setup to block PRs or even trigger on PRs. Once merged in, I will add the webhook to run it. After running with it for a while we can enable the merge blocking etc.

**Comment by wesleyw72** [comment]
Here's a build that works: https://buildkite.com/improbable/keanu-premerge/builds/50#81bbc81c-9a53-4ee6-a3a6-5f0ad457bb01

**Pull request merged**